server/utils/hollaex-tools-lib/tools/user.js

```diff
@@ -1628,6 +1628,27 @@ const setUsernameById = (userId, username) => {
 		});
 };
 
+const disableUserWithdrawal = async (user_id, opts = { expiry_date : null }) => {
+	const user = await getUserByKitId(user_id, false);
+	let { expiry_date } = opts;
+
+	if (!user) {
+		throw new Error(USER_NOT_FOUND);
+	};
+
+	let withdrawal_blocked = null;
+
+	if (expiry_date) {
```

Comment:
expiry_date being null indicates that the admin wants to remove the ban for the withdrawal? @fetok12